munimap/export/mapfish.py

```diff
@@ -120,30 +120,30 @@ def replace_icon_src(style):
         replace_icon_src(flatstyle)
 
 
-def mapfish_layers(requested_layers, bbox=None, srs=None, dpi=None, scale=None, icons_dir='', fc_layer_payloads=None, is_custom=False):
+def mapfish_layers(requested_layers, bbox=None, srs=None, dpi=None, scale=None, icons_dir='', fc_layer_payloads=None, opacities=None, is_custom=False):
+    opacities = opacities or dict()
     if len(requested_layers) == 0:
         return []
-    # convert list of strings to needed format
-    if isinstance(requested_layers[0], str):
-        requested_layers = [[name, {}] for name in requested_layers]
     layers = []
 
     # TODO handle layer not found
     num_offset = 1
-    for name, params in reversed(requested_layers):
-        requested_opacity = 1
-        if 'opacity' in params:
-            requested_opacity = float(params['opacity'])
-
-        print_name = '%s_print' % name
+    for name in reversed(requested_layers):
+        print_name = f'{name}_print'
         if print_name in current_app.mapfish_layers:
             name = print_name
 
         layer = deepcopy(current_app.mapfish_layers[name])
 
+        requested_opacity = 1
```

Comment:
what is the benefit of setting this default, if we override it anyway (see two lines below)?